docs/content/en/plugins/introduction.md

```diff
@@ -65,7 +65,7 @@ This is available to our runtime component under `Runtime.metadata`, allowing us
 // that lets us declare route handlers
 export default function BlogRedirectRuntime(Runtime) {
```

Comment:
This could potentially be replaced with a destructuring e.g. `BlogRedirectRuntime({ Router })`